build/Neolution.CodeAnalysis.globalconfig

```diff
@@ -250,6 +250,10 @@ dotnet_diagnostic.S3267.severity = suggestion
 ; Developers should be warned when they create disposable objects without disposing them
 dotnet_diagnostic.CA2000.severity = warning
 
+# S1133	Do not forget to remove this deprecated code someday.
+; Deprecated code should eventually be removed, but it should not break the current build.
+dotnet_diagnostic.CA2000.severity = suggestion
```

Comment:
maybe is should be the S1133 like in the comment?

Reply:
yeah... good point 😆